advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/getting_started/installing/index.mdx

```diff
@@ -1,30 +1,31 @@
 ---
-navTitle: Installing EDB DMS Reader
-title: Installing EDB DMS Reader on Linux
+navTitle: Installing
+title: Installing EDB Data Migration Service Reader on Linux
+
```

Comment:
Is it possible to add the following to the frontmatter of this index.mdx?  
```suggestion
indexCards: none

```
That way, we don't have both a Table of Content, and navigation tiles. That way, we can avoid unnecessary repetitions. 
<img width="825" alt="Screenshot 2024-09-24 at 12 39 03 copy" src="https://github.com/user-attachments/assets/89ef123c-2c2d-4d91-85aa-a0890f4068dc">

Reply:
This is a good idea. However, if I make this change it will apply to all products. I would probably make this change in a separate PR because it's going to affect a hundred topics or so. @nidhibhammar what do you think of this idea?

Reply:
Okay, that is a good idea. Let me open a Jira for this work https://enterprisedb.atlassian.net/browse/DOCS-1040